vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/ProjectNavigationBarComponent.java

```diff
@@ -58,39 +60,48 @@ private void initNavigationBar() {
     navigationBarLayout = new HorizontalLayout();
     //Todo Generate Custom Component containing Button, statusIndicator and label
     VerticalLayout projectInformationLayout = new VerticalLayout();
-    projectInformationButton = new Button(VaadinIcon.CLIPBOARD_CHECK.create());
+    Icon projectInformationIcon = VaadinIcon.CLIPBOARD_CHECK.create();
+    projectInformationIcon.setSize(IconSize.LARGE);
+    projectInformationButton = new Button(projectInformationIcon);
     Label projectInformationLabel = new Label("Project Information");
     projectInformationLayout.add(projectInformationButton, projectInformationLabel);
     styleNavLabelLayout(projectInformationLayout);
 
     VerticalLayout experimentalDesignLayout = new VerticalLayout();
-    Label experimentalDesignLabel = new Label("Experimental Design");
-    experimentalDesignButton = new Button(VaadinIcon.SITEMAP.create());
-    experimentalDesignLayout.add(experimentalDesignButton, experimentalDesignLabel);
+    Label experimentsLabel = new Label("Experiments");
+    Icon experimentsIcon = VaadinIcon.SITEMAP.create();
+    experimentsButton = new Button(experimentsIcon);
+    experimentsIcon.setSize(IconSize.LARGE);
+    experimentalDesignLayout.add(experimentsButton, experimentsLabel);
     styleNavLabelLayout(experimentalDesignLayout);
 
     VerticalLayout samplesLayout = new VerticalLayout();
-    samplesButton = new Button(VaadinIcon.FILE_TABLE.create());
+    Icon sampleIcon = VaadinIcon.FILE_TABLE.create();
+    sampleIcon.setSize(IconSize.LARGE);
+    samplesButton = new Button(sampleIcon);
     Label samplesLabel = new Label("Samples");
     samplesLayout.add(samplesButton, samplesLabel);
     styleNavLabelLayout(samplesLayout);
 
     VerticalLayout rawDataLayout = new VerticalLayout();
-    rawDataButton = new Button(VaadinIcon.CLOUD_DOWNLOAD.create());
+    Icon rawDataIcon = VaadinIcon.CLOUD_DOWNLOAD.create();
+    rawDataIcon.setSize(IconSize.LARGE);
+    rawDataButton = new Button(rawDataIcon);
     Label rawDataLabel = new Label("Raw Data");
     rawDataLayout.add(rawDataButton, rawDataLabel);
     styleNavLabelLayout(rawDataLayout);
 
     VerticalLayout resultsLayout = new VerticalLayout();
-    resultsButton = new Button(VaadinIcon.SEARCH.create());
+    Icon resultsIcon = VaadinIcon.SEARCH.create();
+    resultsIcon.setSize(IconSize.LARGE);
+    resultsButton = new Button(resultsIcon);
```

Comment:
search or result?

Reply:
Search is just the look of the icon we're currently employing in for the resultButton in the navigationBar context. Since this could change I'd keep the iconName consistent to mean the navigationcontext in which the icon will be used

Reply:
Ah the page with the data :) nvm